src/components/Clients/ClientListTableRow.jsx

```diff
@@ -48,16 +51,11 @@ const ClientListTableRow = ({ labelId, client, state, dispatch }) => {
 
   // Event handler for deleting client from client list
   const handleDeleteClient = async () => {
-    if (
-      !window.confirm(
-        `You're about to delete ${client.person} from your client list, do you wish to continue?`
-      )
-    ) {
-      return;
-    }
+   
     runNotification(`Deleting "${client.person}" from client list...`, 3, state, dispatch);
     await removeUser(client);
     runNotification(`"${client.person}" deleted from client list...`, 3, state, dispatch);
+    setShowClientDeleteModal(false);
```

Comment:
Overall, I think this works great. Functionally, it works as intended as well as a confirmation modal.

https://github.com/codeforpdx/PASS/assets/14917816/0a743977-d5a0-4332-bc7f-f43c724b19b5

UI-wise, think we should move the the set function (setShowClientDeleteModal) to the top and perhaps with time of 7s instead of 3s for the first runNotification function in Line 55. That way we don't have to be stuck in the modal until it finishes removeUser and the user could do other things while it's being removed.

https://github.com/codeforpdx/PASS/assets/14917816/bd3c0b0a-9a46-447b-bfd4-46822c5c15e6